src/db.ts

```diff
@@ -28,42 +51,134 @@ interface InternalReminder {
 	ping: string | null;
 }
 
-// -- Activity Tracker Database --
-export const activityTrackerBlacklistDb = new Enmap<string[]>("trackerBlacklist");
-activityTrackerBlacklistDb.ensure("general-user", []);
-activityTrackerBlacklistDb.ensure("general-game", []);
-
-// key: "[user id]-[game]"
-export const activityTrackerLogDb = new Enmap<ActivityLogEntry[], InternalActivityLogEntry[]>({
-	name: "trackerLog",
-	serializer: (data) => data.map((e) => ({ duration: e.duration.toISO(), date: e.date.toISO() })),
-	deserializer: (data) =>
-		data.map((e) => ({
-			duration: Duration.fromISO(e.duration),
-			date: DateTime.fromISO(e.date, { setZone: true }),
-		})),
-});
+/*
+--------------------
+- Tracker Database -
+--------------------
+*/
 
-export interface ActivityLogEntry {
-	duration: Duration;
-	date: DateTime;
+/** Interface for tracker logs */
+export interface TrackerLog {
+	/** id of the log */
+	id: string;
+	/** When a log got logged */
+	date: Date;
```

Comment:
ISODate

Reply:
Oh yeah thats not the internal interface. This should be a DateTime date

Reply:
well, Ive only used .date with a .getTime() after it, so I desterilize with .getTime() now.
So that cleans up some recursive code again.
*i really dont want to use Datetime (idk why, its just a thing)*